libs/features/personalization/personalization.js

```diff
@@ -246,6 +248,26 @@ export function parseConfig(data) {
       variants[vn] = { commands: [] };
     });
 
+    const laterToDate = (laterExpression) => {
+      if (!laterExpression) return null;
+      const regex = /at (\d{1,2}):(\d{1,2}) on the (\d{1,2}) day of (\w+) in (\d{4})/;
+      const matches = laterExpression.match(regex);
+      if (matches && matches.length === 6) {
```

Comment:
what is the purpose of `matches.length === 6`? to access matches[X] with a peaceful mind? normally regex matching is enough as it implies 5 matching groups.

Reply:
true, its redundant, will remove it